### DISCUSSION TITLE
Cannot upload proof of academic status

### DISCUSSION THREAD

**Discussion by xhos** [community/community#67405] [GitHub Education]
### Hi everyone,

Have been trying to upload proof for days now, but after the I click the button "process my application" absolutely nothing happens. It just starts processing and then the page just resets. My document is perfectly good, I even printed it out and showed it into a camera but that did nothing. Please help.

### A suggestion...(optional)

Fix the page

**Comment by MaryaBaloch20** [comment]
If you're experiencing issues with the GitHub Education application not processing your documents, maybe you can try any of these solutions:

1. Sometimes, issues like this can be browser-specific. Try accessing the GitHub Education application on different web browsers (e.g., Chrome, Firefox, Edge) to see if the problem persists.

2. Clear your browser's cache and cookies. Cached data can sometimes interfere with web forms and submissions. After clearing the cache, try the application process again.

3. Browser extensions or add-ons can sometimes interfere with website functionality. Temporarily disable any extensions you have installed and try submitting your application again.

4. Ensure that your document is in an acceptable format and meets the requirements specified by GitHub Education. Common formats include PDF, PNG, or JPEG. Double-check that the file size is not too large.

5. If none of the above solutions work, consider reaching out to GitHub Support for assistance. They can provide specific guidance and help you troubleshoot the issue further.

6. Sometimes, technical issues on the platform's side can cause delays or problems with form submissions. If you've tried all of the above steps and are still facing issues, you may need to wait for a while and try again later.

**Comment by nlahary** [comment]
Hey, did you manage to solve it ?

**Reply by xhos** [reply]
I did, don't remember how tho. Good luck :)
